### ISSUE TITLE
Make Librum narrator / screen reader compatible

### ISSUE THREAD

**Issue by vazsingh** [Librum-Reader/Librum#125]
**Is your feature request related to a problem? Please describe.**
I have been trying to use the native Windows Narrator (screen reader) to read a .pdf ebook in Librum but it cannot read the text. 

**Describe the solution you'd like**
It would be great if Librum could present PDF ebooks in a way that Windows Narrator (or other screen reader assistants) are able to recognise the text.

**Describe alternatives you've considered**
N/A

**Additional context**
Windows 11, Windows Narrator.

**Comment by DavidLazarescu** [comment]
Thanks for the feature request. I have never used the windows screen narrator before and don't know the requirements for it to read the screen. We will be adding TTS (Text to speech), thus automatic narration to Librum soon though. I think this should cover your use case of the windows screen narrator.

We would of course happily take any PRs regarding this.

**Comment by vazsingh** [comment]
That would be fantastic and likely solve the issue. I'm happy for this issue to be closed unless you wish to keep it open until the feature is implemented. Thank you

**Comment by distbit0** [comment]
Can't wait for tts support! :) This will be epic
